package.json

```diff
@@ -9,31 +9,49 @@
     "build:min": "mkdir -p build && MDL_ENV=production webpack -p",
     "dist": "npm run clean && npm run build && npm run build:min",
     "dev": "npm run clean && MDL_ENV=development webpack-dev-server --content-base demos --inline --hot",
-    "lint:js": "eslint --fix packages webpack.config.js",
-    "postinstall": "lerna bootstrap"
+    "lint:js": "eslint --fix packages test webpack.config.js karma.conf.js",
```

Comment:
I have it set up to run as the "pretest" script as well, so it'll execute
when "npm test" is run. However developers can also manually run it if they
wish.
On Wed, Jul 20, 2016 at 09:10 Sérgio Gomes <notifications@github.com> wrote:

> In package.json
> <https://github.com/google/material-design-lite/pull/4567#discussion_r71520727>
> :
>
> > @@ -9,31 +9,49 @@
> >      "build:min": "mkdir -p build && MDL_ENV=production webpack -p",
> >      "dist": "npm run clean && npm run build && npm run build:min",
> >      "dev": "npm run clean && MDL_ENV=development webpack-dev-server --content-base demos --inline --hot",
> > -    "lint:js": "eslint --fix packages webpack.config.js",
> > -    "postinstall": "lerna bootstrap"
> > +    "lint:js": "eslint --fix packages test webpack.config.js karma.conf.js",
>
> This one's meant to be manually used by developers, not as part of some
> build process, right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/material-design-lite/pull/4567/files/e26e3d3de0cf7001a07c785f7cb5ed0beef0b20e#r71520727>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABIV9dlHIdmCOue2CQlArCAzXAhGlvDoks5qXh5jgaJpZM4JQBo4>
> .
>